graylog2-server/src/main/java/org/graylog2/indexer/searches/Searches.java

```diff
@@ -172,6 +174,7 @@ public Searches(Configuration configuration,
 
         this.esRequestTimer = metricRegistry.timer(name(Searches.class, "elasticsearch", "requests"));
         this.esTimeRangeHistogram = metricRegistry.histogram(name(Searches.class, "elasticsearch", "ranges"));
+        this.esTotalSearchesCounter = metricRegistry.counter(name(Searches.class, "elasticsearch", "searches"));
```

Comment:
The metric name should be more explicit, e. g. `total-searches` instead of `searches`.